osquery/events/darwin/event_taps.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <ApplicationServices/ApplicationServices.h>
+
+#include <osquery/flags.h>
+
+#include "osquery/events/darwin/event_taps.h"
+
+namespace osquery {
+
+/// Flag that turns the eventing system for event taps on or off
+FLAG(bool,
+     disable_event_tapping,
```

Comment:
Instead of default-disable, write this as `enable_keyboard_events` and set the default to false. Then someone who wants to opt-in write `--enable_keyboard_events`.